docker/cgc_metadata_to_qiime_mapping_file.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, Gregory Poore.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# ----------------------------------------------------------------------------
+import numpy as np
+import pandas as pd
+import sevenbridges as sbg
+import glob, sys, getopt, os, json, unicodedata, argparse, time
+from pandas.io.json import json_normalize
+from tqdm import tqdm # progressbar
+from tqdm import trange
+
+parser = argparse.ArgumentParser(description="""
```

Comment:
`argparse` was mostly chosen out of convenience but also because various sources said it was the recommended CLI parsing module in the Python standard library. If it needs to be ported to `click`, it can be done.